.github/workflows/release.yml

```diff
@@ -119,8 +161,8 @@ jobs:
 
   docker-parachain:
     runs-on: ubuntu-latest
-    needs: ["get-variables", "build", "generate-parachain-specs"]
-    if: needs.get-variables.outputs.tag-exists == 0 && github.event_name != 'pull_request'
+    needs: ["build", "generate-parachain-specs"]
+    if: github.event_name == 'push'
```

Comment:
So we still build docker on all internal branches, but not on external PR branches, right?

I think this is good.

Reply:
Yes